### PR TITLE
[Fixes: #12680] Republish image on changing profile settings

### LINKED ISSUE #12680: Can't hide Profile picture from another user if it was once seen
# Bug Report

## Problem 
If first UserA sets `Show your profile picture to: Contacts` and UserB is a contact of UserA, but after that UserA changes this setting to `Show your profile picture to: None`, then UserB will still see the profile picture of UserB.
But if UserA first changes the settings to `Show your profile picture to: None`, and then started a chat with UserB, then UserB will not see UserA's profile photo.

#### Expected behavior

- UserB can't see UserA's profile picture after changing the setting or deleting UserB from contacts
- **As a temporary solution, there may be a warning to users that they will not be able to hide their profile picture for users who have already seen their profile picture (add some explanation to the `Show my picture to` screen** (suggested [here](https://github.com/status-im/status-react/pull/11768#issuecomment-931240752)))

#### Actual behavior

Once seen by another user, the profile picture cannot be hidden

### Reproduction

1. Add profile photo
2. Add UserB to contacts and start 1-1 chat
3. UserB adds UserA to contacts
4. UserA: Settings > Privacy and security > `Show your PP to: None` (or delete UserB from contacts)
5. UserB: check profile photo of UserA

### Additional Information

- Status version: nightly 4/10
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by cammellos**
Fixes: #12680

status: ready

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]

File: translations/en.json [hunk 1]
```diff
@@ -1025,6 +1025,7 @@
     "privacy-photos": "Profile Photo Privacy",
     "privacy-and-security": "Privacy and security",
     "privacy-policy": "Privacy policy",
+    "privacy-show-to": "People who have already seen your profile picture will continue to do so",
```
Comment (Samyoul) [inline translations/en.json]: I wonder if a name like below would be more informative in the future.

```suggestion
    "privacy-show-to-warning": "People who have already seen your profile picture will continue to do so",
```
or

```suggestion
    "privacy-show-to-title": "People who have already seen your profile picture will continue to do so",
```

**Review by Samyoul — APPROVED** [review]

**Comment by qoqobolo** [comment]
@cammellos thanks for adding this!

#### ISSUE 1: No way to see the full text of the warning

Can we add the full text to the setting screen below the options `Everyone`, `Contacts`, `None`?

Screenshot 2021-10-11 at 11 09 26


#### ISSUE 2: Options `Contacts` and `Everyone` are shortned in the middle

Maybe, it would look better if we display these words in full and slightly reduce the width of the warning text preview

Screenshot 2021-10-11 at 11 00 13

**Comment by cammellos** [comment]
@qoqobolo it should be ready to be retested, thank you!

**Comment by qoqobolo** [comment]
@cammellos you mean the latest build `8767dee`, right? It has already been retested :)
Looks good, PR can be merged.

**Comment by cammellos** [comment]
@qoqobolo thank you! merging

**Pull request closed without merging**